examples/react-native/__tests__/js/quickstart/quickstart-find-sort-filter.test.jsx

```diff
@@ -6,7 +6,7 @@ import {createRealmContext} from '@realm/react';
 import {Button} from 'react-native';
 import {render, fireEvent, waitFor} from '@testing-library/react-native';
 let higherOrderProfileName;
-let primaryKey;
+const YOUR_PRIMARY_KEY= new Realm.BSON.ObjectId();
```

Comment:
Is prettier/format on save working?
```suggestion
const YOUR_PRIMARY_KEY = new Realm.BSON.ObjectId();
```